src/renderer/geometry/instanced_mesh.rs

```diff
@@ -90,24 +130,189 @@ impl InstancedMesh {
     /// Update the instances.
     ///
     pub fn set_instances(&mut self, instances: &Instances) {
+        // For code review; should this be here > I dev with --release, that hides this and then
+        // it fails elsewhere.
```

Comment:
So, this is something I ran into several times now, I always build with `--release`, because otherwise everything is slow, but that hides this assertion. Then it fails elsewhere, without a nice convenient print that's like; your color vector isn't equal size. What's the actual cost of doing this assertion and panicking with a nice message?

Though, now that I think of it it's probably possible to compile with optimisations and debug assertions. [Yes](https://doc.rust-lang.org/cargo/reference/profiles.html#debug-assertions), may be worth documenting somewhere?